report/11_Systems-Design.md

```diff
@@ -1 +1,11 @@
-Design of your ITU-MiniTwit systems
+A component digram describing the flow of data through the MiniTwit system itself can be seen in the figure below.  
+
+![Component diagram](./images/component_diagram.png)
+
+The MiniTwit application has two main C# Projects, MiniTwit.Web.App, containing the web app, and MiniTwit.Models containing the repositories for the different types of data handled by the application. The application also has other projects containing the MiniTwitContext connecting to the Database, or containing utility functions, but these are not illustrated in the diagram above. 
```

Comment:
Why is Models and Web.App the main projects? Entities is just as important? We also have the Flag app.